test_otp.qmd

```diff
@@ -94,10 +112,19 @@ log1 <- otp_build_graph(otp = path_otp,router = "Leeds", dir = path_data,memory
 We initialise the OTP server once the graph has been built
 ```{r}
 #| label: init-server
+#| eval: false
 log2 <- otp_setup(otp = path_otp, dir = path_data,router = "Leeds",memory = 15e3)
-otpcon <- otp_connect(timezone = "Europe/London",router = "Leeds")
 ```
 
+Connect to the server as follows:
+
+```{r}
+otpcon <- otp_connect(timezone = "Europe/London", router = "Leeds")
```

Comment:
Heads-up @mem48 this generates an error message:

```
Error in otp_connect(timezone = "Europe/London", router = "Leeds") : 
  Router http://localhost:8080/otp/routers/Leeds does not exist. Valid routers are: default
```

Reply:
That's because you have used the wrong router name, as the message suggest the valid options are "default"

Reply:
Can you get it running @mem48 and confirm? Latest error I've hit here:

```
10:21:19.815 INFO (NetworkListener.java:790) Started listener bound to [0.0.0.0:8080]
10:21:19.821 INFO (NetworkListener.java:790) Started listener bound to [0.0.0.0:8081]
10:21:19.824 INFO (HttpServer.java:278) [HttpServer] Started.
10:21:19.824 INFO (GrizzlyServer.java:148) Grizzly server running.
10:21:35.819 ERROR (PlannerResource.java:97) System error
java.lang.NullPointerException: zone
```

Reply:
https://github.com/ITSLeeds/routingday/blob/main/test_otp.qmd